core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -1051,6 +1051,59 @@
     sql(mv, query).ok();
   }
 
+  @Test void testAggCallArgExpressedByMvProjs1() {
+    final String mv = ""
+        + "select \"deptno\", \"salary\""
+        + "from \"emps\" group by \"deptno\", \"salary\"";
+    final String query = ""
+        + "select \"deptno\", sum(\"salary\")"
+        + "from \"emps\" group by \"deptno\"";
+    sql(mv, query).ok();
```

Comment:
This is not correct, should not match in this case.

Reply:
Yeah, You are right.
It's incorrect test case.
I have updated pr, please review it again.
Thanks a lot.